DQMOffline/Trigger/python/PhotonMonitor_cff.py

```diff
@@ -113,61 +127,73 @@
-
-DiphotonMass55NewANDnoPV_monitoring = hltPhotonmonitoring.clone()
+DiphotonMass90_monitoring = hltPhotonmonitoring.clone(
+    FolderName = 'HLT/HIG/DiPhoton/diphotonMass90/',
+    nphotons = 2,
+    photonSelection = "(pt > 20 && abs(eta)<1.4442 && hadTowOverEm<0.12 && full5x5_sigmaIetaIeta()<0.015 && full5x5_r9>.5)||(pt > 20 && abs(eta)<2.5 && abs(eta)>1.5556 && hadTowOverEm<0.12 && full5x5_sigmaIetaIeta()<0.035 && full5x5_r9>.8)",
+    numGenericTriggerEventPSet = dict(hltPaths = ["HLT_Diphoton30_22_R9Id_OR_IsoCaloId_AND_HE_R9Id_Mass90_v*"])
+)
+
+DiphotonMass95_monitoring = hltPhotonmonitoring.clone(
+    FolderName = 'HLT/HIG/DiPhoton/diphotonMass95/',
+    nphotons = 2,
+    photonSelection = "(pt > 20 && abs(eta)<1.4442 && hadTowOverEm<0.12 && full5x5_sigmaIetaIeta()<0.015 && full5x5_r9>.5)||(pt > 20 && abs(eta)<2.5 && abs(eta)>1.5556 && hadTowOverEm<0.12 && full5x5_sigmaIetaIeta()<0.035 && full5x5_r9>.8)",
+    numGenericTriggerEventPSet = dict(hltPaths = ["HLT_Diphoton30_22_R9Id_OR_IsoCaloId_AND_HE_R9Id_Mass95_v*"])
+)
+DiphotonMass55AND_monitoring = hltPhotonmonitoring.clone(
+    FolderName = 'HLT/HIG/DiPhoton/diphotonMass55AND/',
+    nphotons = 2,
+    photonSelection = "(pt > 20 && abs(eta)<1.4442 && hadTowOverEm<0.12 && full5x5_sigmaIetaIeta()<0.015 && full5x5_r9>.5)||(pt > 20 && abs(eta)<2.5 && abs(eta)>1.5556 && hadTowOverEm<0.12 && full5x5_sigmaIetaIeta()<0.035 && full5x5_r9>.8)",
+    numGenericTriggerEventPSet = dict(hltPaths = ["HLT_Diphoton30PV_18PV_R9Id_AND_IsoCaloId_AND_HE_R9Id_PixelVeto_Mass55_v*"]),
+    histoPSet = dict(massBinning = [50.,51.,52.,53.,54.,55.,56.,57.,58.,59.,60.,61.,62.,63.,64.,65.,66.,67.,68.,69.,70.,75.,80.,90.,110.,150.])
+)
+
+
+DiphotonMass55EB_monitoring = hltPhotonmonitoring.clone(
+    FolderName = 'HLT/HIG/DiPhoton/diphotonMass55EB/',
+    nphotons = 2,
+    photonSelection = "(pt > 20 && abs(eta)<1.4442 && hadTowOverEm<0.12 && full5x5_sigmaIetaIeta()<0.015 && full5x5_r9>.5)",
+    numGenericTriggerEventPSet = dict(hltPaths = ["HLT_Diphoton30EB_18EB_R9Id_OR_IsoCaloId_AND_HE_R9Id_PixelVeto_Mass55_v*"]),
+    histoPSet = dict(massBinning = [50.,51.,52.,53.,54.,55.,56.,57.,58.,59.,60.,61.,62.,63.,64.,65.,66.,67.,68.,69.,70.,75.,80.,90.,110.,150.])
+)
+
+DiphotonMass55ANDnoPV_monitoring = hltPhotonmonitoring.clone(
+    FolderName = 'HLT/HIG/DiPhoton/diphotonMass55ANDnoPV/',
+    nphotons = 2,
+    photonSelection = "(pt > 20 && abs(eta)<1.4442 && hadTowOverEm<0.12 && full5x5_sigmaIetaIeta()<0.015 && full5x5_r9>.5)||(pt > 20 && abs(eta)<2.5 && abs(eta)>1.5556 && hadTowOverEm<0.12 && full5x5_sigmaIetaIeta()<0.035 && full5x5_r9>.8)",
+    numGenericTriggerEventPSet = dict(hltPaths = ["HLT_Diphoton30PV_18PV_R9Id_AND_IsoCaloId_AND_HE_R9Id_NoPixelVeto_Mass55_v*"]),
+    histoPSet = dict(massBinning = [50.,51.,52.,53.,54.,55.,56.,57.,58.,59.,60.,61.,62.,63.,64.,65.,66.,67.,68.,69.,70.,75.,80.,90.,110.,150.])
```

Comment:
This line hence is correct now, see my comment above in this file

Reply:
Thank you @jfernan2 for your comment. So, there is no need to change anything in this regard, I think!